src/components/ui/Modal/modal.module.scss

```
@@ -0,0 +1,50 @@
+@import "src/styles/scaffold";
+
+.fs-modal {
+  // --------------------------------------------------------
+  // Design Tokens for Modal
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-modal-position-top                          : 30vh;
+  --fs-modal-position-right                        : var(--fs-spacing-4);
+  --fs-modal-position-left                         : var(--fs-spacing-4);
+  --fs-modal-max-width                             : calc(var(--fs-grid-breakpoint-desktop) / 3);
+  --fs-modal-min-height                            : 30px;
+  --fs-modal-margin                                : auto;
+
+  // Tablet properties
+  --fs-modal-width-tablet                          : calc(100vw / 3);
+  --fs-modal-min-width-tablet                      : calc(var(--fs-grid-breakpoint-desktop) / 3);
+
+  // Transition properties
+  --fs-modal-transition-state-in-timing            : 0.2s;
+  --fs-modal-transition-state-in-property          : transform;
+  --fs-modal-transition-state-in-function          : ease-in;
+  --fs-modal-transition-state-out-timing           : 0.2s;
+  --fs-modal-transition-state-out-property         : transform;
+  --fs-modal-transition-state-out-function         : ease-in;
```

I believe here we can use different values only for the function tokens, and for the others we can use the global ones:

```suggestion
  --fs-modal-transition-state-timing              : var(--fs-transition-timing);
  --fs-modal-transition-state-property          : var(--fs-transition-property);
  --fs-modal-transition-state-in-function      : ease-in;
  --fs-modal-transition-state-out-function    : ease-out;
```